api/envoy/api/v2/route/route_components.proto

```diff
@@ -150,6 +150,14 @@ message VirtualHost {
   // independently (e.g.: values are not inherited).
   RetryPolicy retry_policy = 16;
 
+  // [#not-implemented-hide:]
+  // Implementation specific configuration which depends on the implementation being instantiated.
```

Comment:
this description is pretty confusing for someone that doesn't know how the extensions usually work, maybe it should mention that this also defined what implementation is being instantiated?

Reply:
I'm confused. `RetryPolicy` already has a typed config `Any`. Surely any implementation-specific extension to this should be nested in there?

Reply:
> RetryPolicy already has a typed config Any

I am not sure if this is true. I didn't find the field with typed `Any` https://github.com/envoyproxy/envoy/blob/d1a36f1ea24b38fc414d06ea29c5664f419066ef/api/envoy/api/v2/route/route_components.proto#L981-L1075